src/components/Card/index.js

```diff
@@ -1,18 +1,26 @@
 // @flow
 import React, { Component } from 'react'
+import type { Key } from 'react'
 
 import CardWrap from './CardWrap'
 import CardImage from './CardImage'
 import CardContent from './CardContent'
 
-const getUrlPath = data => typeof data === 'object' ? data.url : data
+import { getUrlPath } from '../utils'
+
+export type CardSizes = 'large' | 'small'
 
 type CardProps = {
+  className?: string,
   contrast?: boolean,
-  endpoint?: string,
-  url: string,
   endpoint: string,
```

Comment:
this is optional. The only required paramter is `url`

Reply:
This is from flow @Kikobeats https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

Reply:
👍